src/services/loginService.tsx

```diff
@@ -0,0 +1,6 @@
+import axios, { AxiosResponse } from 'axios'
+
+import { LoginCredentials } from '../interfaces/LogInCredentials'
+import { LoginResponse } from '../interfaces/LogInResponse'
+
+export const logIn = async (credentials: LoginCredentials ) : Promise<AxiosResponse<LoginResponse>> => axios.post<LoginResponse>('https://library-practice-app.azurewebsites.net/api/Auth/login', credentials)

```

Comment:
sta nam je sa refreshovanjem tokena?

Reply:
nisam to koristila jer nam token traje godinu dana ako se dobro secam

Reply:
ali mogu dodati za opste slucajeve